authors/apps/authentication/views.py

```diff
@@ -73,3 +89,74 @@ def update(self, request, *args, **kwargs):
 
         return Response(serializer.data, status=status.HTTP_200_OK)
 
+@api_view(['GET', 'POST'])
+def password_reset(request):
```

Comment:
@JSnakegitHub , I like the fact that you decoupled the password reset functionality into its own app. That means the `PasswordResetToken `app can manage its views instead of having them here in the authentication app.

Reply:
Hey Roland . do you think he will be able to make these changes without breaking the code? I suggest we look into this after the demo, or maybe he could create another branch for these changes in order to leave this branch safe.

Reply:
Roland, I agree with Sue, I will try to refactor this after the demo